src/uct/ib/ud/base/ud_iface.h

```diff
@@ -383,6 +386,23 @@ uct_ud_iface_dispatch_zcopy_comps(uct_ud_iface_t *iface)
     uct_ud_iface_dispatch_async_comps_do(iface);
 }
 
+static UCS_F_ALWAYS_INLINE
+void uct_ud_iface_async_handler(uct_worker_h worker, uint8_t *elem_on,
```

Comment:
1. the function name should be more informative - e.g uct_ud_iface_async_remove_cb_enable()
  also, no need to pass worker+flag+elem - you already have all this info in the iface
1. can make function code shorter:
```
if (do_reg && !iface->async.elem_on) {
  ...
} else if (!do_reg && iface->async.elem_on) {
  ...
}

```